ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/snmp-settings.j2

```diff
@@ -22,6 +22,31 @@
 {%         endfor %}
 {%     endif %}
 
+{% if snmp_server.communities is arista.avd.defined %}
```

Comment:
@melkypie - we are still in process of refactoring our documentation templates to follow a certain style. 
We plan on publishing a guide soon to give better guidance for external contributors. 

Can you update the jinja2 template to follow this the style from the following example:

- https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2
- https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/documentation/fabric-documentation.j2